lib/sidekiq_adhoc_job/web/templates/jobs/show.html.erb

```diff
@@ -57,16 +57,8 @@
 </form>
 
 <% if @presented_job.require_confirm %>
-  <script>
-    document.getElementById('adhoc-jobs-submit-form').addEventListener('submit', (event) => {
-      const confirmPrompt = "<%= @presented_job.confirm_prompt_message %>";
-      const input = prompt(`Please enter "${confirmPrompt}" to confirm`);
-      if (input !== confirmPrompt) {
-        event.preventDefault();
-        if (input !== null) {
-          alert('Your confirmation input is incorrect, try again.');
-        }
-      }
-    })
+  <div id="presented-job" data-confirm-prompt="<%= @presented_job.confirm_prompt_message %>">
+  </div>
```

Comment:
not a big deal but I'd argue this `data-confirm-prompt` could have just been on the `<form>` instead